src/common/helpers/request.ts

```diff
@@ -39,19 +38,6 @@ client.interceptors.response.use(
     return response;
   },
   (error: AxiosError<ValidationBag>) => {
-    if (
```

Comment:
@beganovich Is this logic only for the /quota endpoint?

Reply:
No, we use it for all einvoice endpoints. Can you please do double-checks to make sure we exclude quota?

Reply:
@beganovich Okay, so I ensured that the toaster for the /quota endpoint is no longer displayed.